CMakeLists.txt

```diff
@@ -616,6 +616,8 @@ if (WITH_OPENH264 AND NOT OPENH264_FOUND)
 endif()
 set(WITH_OPENH264 ${OPENH264_FOUND})
 
+option(WITH_MS_RDPECAM_CLIENT "Build with video camera redirect client" OFF)
```

Comment:
please move to `channels/rdpecam/client/CMakeLists.txt`

Reply:
Done